### PR TITLE
Add 5-47

### PR DESCRIPTION
**Pull request by weilixu**
Completed 5-47

### CODE REVIEW

**Review by yunjoonjung-PNNL — APPROVED** [review]

File: rct229/rules/section5/section5rule47.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    class BuildingRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section5Rule47.BuildingRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, False, True),
+                required_fields={"$..zones[*]": ["surfaces"]},
+            )
+
+        def get_calc_vals(self, context, data=None):
+            building_p = context.proposed
```
Comment (charliepnnl) [inline rct229/rules/section5/section5rule47.py:44]: Only the baseline is used in this rule, so all "p" subscripts should be "b"s.

File: rct229/rules/section5/section5rule47.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        )
+
+    def create_data(self, context, data=None):
+        rmr_proposed = context.proposed
+        return {"climate_zone": rmr_proposed["weather"]["climate_zone"]}
+
+    class BuildingRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section5Rule47.BuildingRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, False, True),
```
Comment (charliepnnl) [inline rct229/rules/section5/section5rule47.py:39]: This rule only uses the baseline, so the arguments should be `(False, True, False)`.

**Review by charliepnnl — APPROVED** [review]
Approved with comments.

File: rct229/rules/section5/section5rule47.py (hunk 1 above)
Comment (weilixu) [reply rct229/rules/section5/section5rule47.py:44]: Not sure why I did proposed...you are correct, it should baseline.

**Pull request merged**